Src/BootCamp.Chapter/CsvLineParser.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal class CsvLineParser
```

Comment:
This class is only here because it was my understanding that students were expected to implement their CSV parser, not use a NuGet package.

Reply:
Yes